Docs/dutch 3.0 (#5005)
open-formulieren/open-forms: docs/changelog-nl.rst

* [:backend:`4969`] Verbeterde UX van de formulier-editor:  

    - Het tabblad basisconfiguratie groepeert nu gerelateerde velden en maakt het overzichtelijker door ze samen te 
      vouwen.  
    - Configuratie van de introductiepagina verduidelijkt ten opzichte van de velden voor introductietekst.  
* Registratie-plugins:  

    * [:backend:`4686`] Alle configuratie-opties voor registratie-plugins worden nu consequent beheerd in een modaal 
      met verbeterde UX.  

    * E-mail:  

        * [:backend:`4650`] De e-mailregistratie-plugin ondersteunt nu het instellen van de ontvanger met 
          formulier-variabelen.  
    * Objecten API:  

        * [:backend:`4978`] De configuratie van "variabelen-mapping" is nu de standaardinstelling - dit heeft geen 
          invloed op bestaande formulieren.  
        * Technische configuratiedocumentatie voor Objecten API bijgewerkt.  
        * [:backend:`4398`] U kunt nu een bestaand gerefereerd object bijwerken in plaats van een nieuw record aan te 
          maken.  
          Bij het bijwerken van het object wordt het BSN van de geauthenticeerde gebruiker geverifieerd aan de hand van 
          de bestaande objectgegevens.  

>         * [:backend:`4418`] U kunt nu individuele delen van het component "addressNL" koppelen.  

Reviewer: ```suggestion
        * [:backend:`4418`] Je kan nu individuele onderdelen van het component "addressNL" koppelen.
```

    * ZGW API's:  

        * [:backend:`4606`] Verbeterde gebruikerservaring van de plugin:  

          - Alle dropdowns/comboboxen hebben nu een zoekveld.  
          - U kunt nu selecteren welke catalogus moet worden gebruikt, zodat alleen relevante zaak- en documenttypen 
            worden weergegeven.  
          - Tijdens de registratie selecteert de plugin automatisch de juiste versie van een zaak- en documenttype.  
          - URL-gebaseerde configuratie kan nog steeds worden gebruikt, maar wordt in de toekomst verwijderd.  

>         * [:backend:`4796`] U kunt nu een product selecteren dat op de aangemaakte zaak wordt ingesteld, vanuit het 
>           geselecteerde zaaktype in de ZGW API's registratie-plugin.  

Reviewer: ```suggestion
        * [:backend:`4796`] Je kan nu een product uit het geselecteerde zaaktype kiezen dat op de aangemaakte zaak wordt ingesteld.
```

        * [:backend:`4344`] U kunt nu selecteren welke Objecten API-groep moet worden gebruikt in plaats van 
          "de eerste" standaard te gebruiken.  
    * StUF-ZDS:  

        * [:backend:`4319`] U kunt nu een aangepaste documenttitel opgeven via de componentconfiguratie.  

>         * [:backend:`4762`] De medeondertekenaar-ID (BSN) wordt nu opgenomen in de aangemaakte zaak.  

Reviewer: ```suggestion
        * [:backend:`4762`] De mede-ondertekenaar-ID (BSN) wordt nu opgenomen in de aangemaakte zaak.  
```
I'm missing consistency in the spelling

* Prefill-plugins:  

    * Documentatie toegevoegd voor product-prefill in gebruikershandleiding.  

    * Objecten API:  

        * [:backend:`4396`, :backend:`4693`, :backend:`4608`, :backend:`4859`] U kunt nu een variabele configureren die 
          vooraf wordt ingevuld vanuit de Objecten API ("product-prefill"):  

          - Het is mogelijk om individuele eigenschappen van het objecttype toe te wijzen aan specifieke 

>             formulier-variabelen.  

Reviewer: ```suggestion
            formuliervariabelen.  
```

>           - Om duplicatie in de configuratie te vermijden, kunt u de configuratie kopiëren van een geconfigureerde 

Reviewer: ```suggestion
          - Om dubbel werk in de configuratie te vermijden, kan je de configuratie kopiëren van een ingestelde 
```

            registratie-backend.  

* Betalingsplugins:  

    * Ogone:  

        * [:backend:`3457`] Aangepaste ``title`` en ``com`` parameters kunnen nu worden gedefinieerd in de Ogone 
          betalingsplugin.  

> * [:backend:`4785`] Bijgewerkte eHerkenning-metadata-generatie om te voldoen aan de nieuwste standaardversie(s).  

Reviewer: ```suggestion
* [:backend:`4785`] De eHerkenning-metadatageneratie is bijgewerkt om te voldoen aan de nieuwste versie(s) van de standaard.
```

* [:backend:`4930`] Het is nu mogelijk om geregistreerde inzendingsmetadata te exporteren via de 
  formulierenstatistieken in de adminpagina. Dit kan worden gebaseerd op een specifieke datumbereik.  

> * [:backend:`2173`] Het kaartcomponent ondersteunt nu het gebruik van een andere achtergrond/tegel-laag.  

Reviewer: ```suggestion
* [:backend:`2173`] Het kaartcomponent ondersteunt nu het gebruik van een andere achtergrond-/tegellaag.  
```

> * [:backend:`4321`] Formulieren kunnen nu een inzendingslimiet hebben. Het SDK toont passende meldingen wanneer deze 
>   limiet is bereikt.  

Reviewer: ```suggestion
* [:backend:`4321`] Formulieren kunnen nu een inzendingslimiet hebben. De UI toont passende meldingen wanneer deze 
  limiet is bereikt.  
```

* [:backend:`4895`] Metadata toegevoegd aan uitgaande bevestigings- en mede-ondertekeningsverzoek-e-mails.  

> * [:backend:`4789`, :backend:`4788`, :backend:`4787`] Toegevoegd: ``django-setup-configuration`` om Open Formulieren 
>   programmatisch te configureren met verbindingsdetails voor Objecten en ZGW API's. U kunt een configuratiebestand 
>   laden via het ``setup_configuration`` management-commando. Aanvullende informatie/instructies zijn te vinden 
>   in :ref:`installation_configuration_cli`.  

Reviewer: ```suggestion
* [:backend:`4789`, :backend:`4788`, :backend:`4787`] ``django-setup-configuration`` is toegevoegd om Open Formulieren 
  programmatisch te configureren met verbindingsparameters voor Objecten- en ZGW-API's. Je kan een configuratiebestand 
  laden via het ``setup_configuration`` management-commando. Aanvullende informatie/instructies zijn te vinden 
  in :ref:`installation_configuration_cli`.  
```

> * [:backend:`4798`] Consistentie toegevoegd aan de bevestigingsbox en verbeterde UX van andere modals.  

Reviewer: ```suggestion
* [:backend:`4798`] De bevestigingsschermen/overlays hebben nu consistent dezelfde UX/UI, en de UX van overige
  modals is verbetered.
```

> * [:backend:`4320`] Verbeteringen aan de mede-ondertekeningsflow en de bijbehorende teksten toegevoegd, evenals meer 
>   flexibiliteit:  

Reviewer: ```suggestion
* [:backend:`4320`] De mede-ondertekeningsflow en de bijbehorende teksten zijn verbeterd en meer flexibel gemaakt:  
```

>     - Specifieke sjablonen voor medeondertekening kunnen nu worden gebruikt voor de inhoud van de bevestigingsschermen, 
>       inclusief de optie om een 'nu medeondertekenen'-knop toe te voegen.  

Reviewer: ```suggestion
    - Er zijn nu sjablonen voor de inhoud van de bevestigingsschermen specifiek voor mede-ondertekening, 
      met de optie om een 'nu mede-ondertekenen'-knop toe te voegen.  
```

>     - Specifieke sjablonen voor medeondertekening kunnen nu worden gebruikt voor de onderwerpregel en inhoud van de 
>       bevestigingse-mail.  

Reviewer: ```suggestion
    - Er zijn nu sjablonen voor de onderwerpregel en inhoud van de bevestigings-e-mail specifiek voor mede-ondertekening.  
```

    - Wanneer links worden gebruikt in de e-mail met mede-ondertekeningsverzoeken, kan de medeondertekenaar nu direct 
      doorklikken zonder een code in te voeren om de inzending te bekijken.  
    - De standaard sjablonen zijn bijgewerkt met betere teksten/instructies.  
    - Vertalingen van verbeterde teksten zijn bijgewerkt.  
* [:backend:`4815`] De minimale verwijderlimiet voor inzendingen is nu 0 dagen, zodat inzendingen op dezelfde dag 
  verwijderd kunnen worden.  

> * [:backend:`4717`] Verbeterde toegankelijkheid voor site-logo, foutboodschap-element en PDF-documenten.  

Reviewer: ```suggestion
* [:backend:`4717`] Verbeterde toegankelijkheid voor site-logo, foutmeldingen en PDF-documenten.  
```

* [:backend:`4719`] Toegankelijkheid verbeterd in postcodevelden.  

> * [:backend:`4707`] Json-Logic-widgets kunnen nu worden aangepast qua grootte.  

Reviewer: ```suggestion
* [:backend:`4707`] JsonLogic-widgets kunnen nu groter gemaakt worden.  
```

* [:backend:`4720`] Toegankelijkheid verbeterd voor de skiplink en het PDF-rapport.  
* [:backend:`4764`] Mogelijkheid toegevoegd om de prijsberekening van inzendingen als variabele in te stellen.  

> * [:backend:`4716`] Vertalingen toegevoegd voor formulier-velden en bijbehorende verbeteringen in foutmeldingen.  

Reviewer: ```suggestion
* [:backend:`4716`] Vertalingen toegevoegd voor formuliervelden en bijbehorende verbeteringen in foutmeldingen.  
```

> * [:backend:`4524`, :backend:`4675`] Selecteren van een formulier-variabele is nu gebruiksvriendelijker. Variabelen 

Reviewer: ```suggestion
* [:backend:`4524`, :backend:`4675`] Selecteren van een formuliervariabele is nu gebruiksvriendelijker. Variabelen 
```

>   worden logisch gegroepeerd en een zoekveld is toegevoegd.  

Reviewer: ```suggestion
  worden logisch gegroepeerd en er is nu een zoekveld.
```

* [:backend:`4709`] Verbeterde foutfeedback bij onverwachte fouten tijdens het opslaan van een formulier in de 
  formulier-editor.  

> **Bugfixes**  

Reviewer: I will continue tomorrow.
